### ISSUE TITLE
interpro  annotation needs improvement

### ISSUE THREAD

**Issue by bradfordcondon** [statonlab/hardwoods_site#119]
- [x] No linkout
http://160.36.205.61:8095/bio_data/2118902

should be easy to fix by editing the DB entry to include a url linkout

- [x] No definition in table

This is probably because our interpro term does not have a definition in the chado.  IE a cvterm/ontology issue.

**Comment by almasaeed2010** [comment]
Stupid question, is there an OBO for INTERPRO that might just solve the no definition issue?

Also, I added the url and url prefix records to `chado.db` but the annotations field is still not displaying a link. Am I missing something here?

**Comment by almasaeed2010** [comment]
Apparently it existed but I can't find an updated version of IPR obo http://obofoundry.org/ontology/ipr.html

**Comment by bradfordcondon** [comment]
* I set the db entries properly to linkout for GO and interpro.
* I updated the annotation field to utlize the linkout

We have an additional problem, where the field looks for the term definition.  We have no defs for these terms.  We only have info in the name, which is not utilized by this field.

https://github.com/tripal/tripal/issues/277

For now, I'm going to output the name instead of the definition to build the table.
